src/kube-runtime/GOPATH/src/github.com/microsoft/runtime/cmd/exithandler/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/microsoft/runtime/pkg/aggregator"
+	"github.com/microsoft/runtime/pkg/logger"
+)
+
+var log *logger.Logger
+
+const defaultExitCode = 255
+
```

Comment:
1. Keep exitcode in your file the same as the process's exit code. If it is hard, remove the one in file, and just use process's exit code.

2. If there is error in your runtime itself, you should not exit  with 255. 255 means runtime unrecognized the code, for runtime itself failure, it can exit anything other than the code in spec. So it is to be:
- code: 256
  phrase: PAIRuntimeExitAbnormally